chronicles/chronos_tools.nim

```diff
@@ -0,0 +1,22 @@
+import
+  chronos,
+  ../chronicles
+
+proc catchOrQuit*(error: Exception) =
+  if error of CatchableError:
```

Comment:
this is not quite right though, chronos only puts catchables in futures, so this function is kind of pointless

Reply:
I would argue that this function is right on a fundamental level. It's an accident that Chronos is putting only Defects in futures right now.

Reply:
https://github.com/status-im/nim-chronos/issues/81#issuecomment-595288809

Reply:
> chronos only puts catchables in futures

And `CancelledError`.